wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1296,6 +1297,15 @@ def workflow_status_with_date(workflow_state):
     return _("%(status_display)s %(task_name)s %(started_at)s") % translation_context
 
 
+@register.inclusion_tag("wagtailadmin/shared/keyboard_shortcuts_dialog.html", takes_context=True)
+def keyboard_shortcuts_dialog(context):
+    user_agent = context["request"].headers.get("User-Agent", "")
+    is_mac = re.search(r"Mac|iPod|iPhone|iPad", user_agent)
+    return {
+        "meta": "⌘" if is_mac else "Ctrl",
```

Comment:
```suggestion
        "mod": "⌘" if is_mac else "Ctrl",
```

Just realised we should be calling this `mod` not `meta` - mod seems to be the convention for things like this. 

The template would also need to be updated.